main_server.py

```
@@ -972,6 +1074,9 @@ async def _warmup_only():
             
             async def _sync_characters_only():
                 """等待预热完成后同步角色卡"""
+                if is_write_fence_active(_config_manager):
+                    logger.info("创意工坊角色卡同步被维护态拦截，延后到后续安全时机")
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)